firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/FunctionCallingConfig.kt

```diff
@@ -17,13 +17,7 @@
 package com.google.firebase.vertexai.type
 
 /**
- * Contains configuration for function calling from the model. This can be used to force function
- * calling predictions or disable them.
- *
- * @param mode The function calling mode of the model
- * @param allowedFunctionNames Function names to call. Only set when the [Mode.ANY]. Function names
- * should match [FunctionDeclaration.name]. With [Mode.ANY], model will predict a function call from
- * the set of function names provided.
+ * The configuration that specifies the function calling behavior.
```

Comment:
Is there a reason we're removing without replacement?

Reply:
We are no longer exposing the enums, and only allowing creating instances of this object via methods. Is that what you mean?

Reply:
I meant the documentation about the functionality being removed, is it fine to just leave that on the functions?

Reply:
@daymxn consolidating the conversation in this thread.

I've added a callout for the factory methods in the companion object as part of the description. I'm unsure of adding a direct call to each of them as it would be easy for the comment and the list of methods to become outdated.

wdyt?